packages/cmf/BREAKING-CHANGES.md

```diff
@@ -0,0 +1,84 @@
+# 1.0 -> 2.0
```

Comment:
This doesn't seem to cover all the breaking changes. Some changes I spotted :
* `bootstrap` that doesn't accept any router conf anymore (history, etc). The README link is not sufficient, it's not a breaking change log explaining the diff.
* `WithExpression HOC` don’t inject context.router anymore

Reply:
I have revert/re-add context.router on WithExpression